src/cljs/ataru/hakija/application_form_components.cljs

```diff
@@ -89,12 +90,16 @@
     (some #(= % "required") (:validators field-descriptor))
     (validator/validate "required" value)))
 
+(defn- add-link-target-prop
+  [text state]
+  [(string/replace text #"<a href=([^>]+)>" "<a target=\"_blank\" href=$1>") state])
+
 (defn- markdown-paragraph
   [md-text]
-  (let [sanitized-html (->> md-text
-                            (md->html)
-                            (.sanitize html-sanitizer)
-                            (.getTypedStringValue))]
+  (let [sanitized-html (as-> md-text v
```

Comment:
I haven't used `as→` before, neat! We could write this as:

~~~clojure
(->> (md->html md-text :custom-transformers [add-link-target-prop])
     (.sanitize html-sanitizer)
     (.getTypedStringValue))
~~~

but I'm fine with using the slightly longer and more explicit form.